### ISSUE TITLE
[solved] please update Slapper MCPE 1.13.0 support

### ISSUE THREAD

**Issue by teracube** [jojoe77777/Slapper#216]
```
2019-12-06 [22:35:38] [Server thread/ERROR]: Could not load plugin 'Slapper': Incompatible network protocol version (plugin requires one of: 361)

2019-12-06 [22:35:39] [Server thread/CRITICAL]: Could not load plugin 'SlapperPlus': Unknown dependency: Slapper
```

**Comment by LeronHinds** [comment]
This plugin does not support forks or spoons of PocketMine.

**Comment by teracube** [comment]
@LeronHinds 
I use latest official release 3.10 v1.13 
https://github.com/pmmp/PocketMine-MP/releases

**Comment by LeronHinds** [comment]
Oh sorry about that, didn't see that it was released today.

**Comment by teracube** [comment]
what is the new protocol version number for 1.13
thanks

**Comment by provsnoobgaming** [comment]
> what is the new protocol version number for 1.13
> thanks
[Google](https://google.com)

**Comment by teracube** [comment]
388 ?

https://minecraft.gamepedia.com/Protocol_version#Bedrock_Edition

**Comment by teracube** [comment]
solved 1b24fa9
thanks @dktapps
